src/plugins/validation/2and3/semantic-validators/parameters-ibm.js

```diff
@@ -92,7 +92,7 @@ module.exports.validate = function({ jsSpec, isOAS3 }, config) {
         let messageAuth =
           'Parameters must not explicitly define `Authorization`.';
         messageAuth = isOAS3
-          ? `${messageAuth} Rely on the \`securitySchemas\` and \`security\` fields to specify authorization methods.`
+          ? `${messageAuth} Rely on the \`securitySchemes\` and \`security\` fields to specify authorization methods.`
```

Comment:
Minor typo Phil pointed out. This should be [securitySchemes](https://swagger.io/docs/specification/authentication/) in OAS3